streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -636,7 +684,10 @@
         "Accepted values are \"" + UPGRADE_FROM_0100 + "\", \"" + UPGRADE_FROM_0101 + "\", \"" +
         UPGRADE_FROM_0102 + "\", \"" + UPGRADE_FROM_0110 + "\", \"" + UPGRADE_FROM_10 + "\", \"" +
         UPGRADE_FROM_11 + "\", \"" + UPGRADE_FROM_20 + "\", \"" + UPGRADE_FROM_21 + "\", \"" +
-        UPGRADE_FROM_22 + "\", \"" + UPGRADE_FROM_23 + "\" (for upgrading from the corresponding old version).";
```

Comment:
> When upgrading from 2.4 to a newer version it is not required to specify this config.

Seems this needs an update?

Reply:
Good catch! Fixed in cd8ba1848cefca1c89b4011e9d0716b8534f6782.